executor/seccomp_policies/func_to_syscall_nrs.h

```diff
@@ -0,0 +1,1253 @@
+#define __arm_setdomainname 121;
```

Comment:
This file seems to exist as executor/func_to_syscall_nrs.h as well. Do we need 2 copies?